source/deployment-options/docker/wazuh-container.rst

```diff
@@ -254,37 +254,108 @@ Change the password of an existing user
 
 .. warning::
 
-   You must add all users created on the Wazuh dashboard to the ``internal_users.yml`` file. If not, executing this procedure deletes them.
+   You must add all users created on the Wazuh dashboard to the ``internal_users.yml`` file. If you don't, executing this procedure deletes them.
```

Comment:
```suggestion
   If you have custom Wazuh indexer users, add them to the ``internal_users.yml`` file. Otherwise, executing this procedure deletes them.
```